integration-tests/js-compute/setup.js

```diff
@@ -11,6 +11,9 @@ const { DICTIONARY_NAME, CONFIG_STORE_NAME, KV_STORE_NAME, SECRET_STORE_NAME } =
   getEnv(serviceName);
 
 function existingStoreId(stores, existingName) {
```

Comment:
in setting up my fastly [secret-store](https://github.com/fastly/js-compute-runtime/blob/main/integration-tests/js-compute/setup.js#L113), it got into a state where it was null and throw an exception.  Not sure if this is necessary or just something I encountered while trying to setup the integration tests.

Reply:
Strange, it would be interesting to dig into the caller of this function, to see what that is happening and potentially resolve the issue at the root.